wolf/ml/referee_ocr/dlib_export.h

```diff
@@ -0,0 +1,5 @@
+#define W_SYSTEM_EXPORT
+
+#if defined(WIN32) || defined(WIN64)
```

Comment:
remove this header and use wolf.hpp from root of wolf. (use W_API instead of W_SYSTEM_EXPORT)